modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-user-provided-vnet-subnet"]
+= Configuring a private storage endpoint on Azure with user-provided VNet and subnet names
+
+Use the following procedure to configure a private storage endpoint on Azure with user-provided VNet and subnet names.
+
+.Prerequisites 
+
+* You have configured the Image Registry Operator for use with Azure.
+
+.Procedure 
+
+. Enter the following command to obtain the VNet and subnet names that are used in the cluster:
++
+[source,terminal]
+----
+$ oc get machinesets -nopenshift-machine-api -ojson|jq '.items[1].spec.template.spec.providerSpec.value | (.vnet, .subnet)'
```

Comment:
When users bring their own vnet and subnets they usually know their names because they configured it themselves (usually before creating the cluster, if I understand it correctly).
To me, this means that they don't need to figure out what the correct vnet and subnet names are. If they don't know their vnet and subnet names, it probably means their cluster is using the installer provisioned vnet and subnets. In this case, leaving the discovery to the operator is the best way to use this feature.

Maybe adding "know the vnet and subnet names" to the prerequisites list for this specific procedure? It might also be worth placing this procedure after the one that uses the operator to discover vnet and subnet names, since the bring your own vnet and subnet is more advanced than the operator discovery procedure. WDY?

Side note: I added this to the testing steps in the PR because it was an easy way to simulate an installation where the user brought their own vnet and subnets.

Reply:
@flavianmissi I switched the procedures to that the other is now first. How's this bullet point?

* You must know the VNet and subnet names used in your Azure environment.

Reply:
I think we want to keep this step. The bullet point sound good for the "bring your own vnet/subnet" configuration!